src/responsive-layout.tsx

```diff
@@ -73,4 +77,29 @@ const ResponsiveLayout: React.FC<Props> = ({children, ...props}) => (
     </InternalResponsiveLayout>
 );
 
+export const ResetResponsiveLayout: React.FC<{
```

Comment:
For now, we keep this new component for internal use. We can export it outside Mistica if we see that there is a real need in the future